dev_guide/builds.adoc

```diff
@@ -11,11 +11,76 @@ toc::[]
 
 == Overview
 A link:../architecture/core_objects/builds.html[build] is a process of creating
-runnable images to be used on OpenShift. There are three types of builds:
+runnable images to be used on OpenShift. There are three build strategies:
 
-- Docker build
-- STI build
-- custom build
+- link:../architecture/core_objects/builds.html#sti-build[Source-To-Image]
+- link:../architecture/core_objects/builds.html#docker-build[Docker]
+- link:../architecture/core_objects/builds.html#custom-build[Custom]
+
+== Defining a buildConfig
+
+A `*buildConfig*` describes a single build definition and a
+set of link:#triggers[triggers] for when a new build should be created.
+
+A buildConfig is a REST object, which can be used in a POST to the API server to
+create a new instance. The following configuration results in a new build
+every time a Docker image tag or the source code changes:
+
+[source.json]
+----
+{
+  "kind": "BuildConfig",
+    "apiVersion": "v1beta3",
+    "metadata": {
+      "name": "ruby-sample-build <1>",
+    },
+  "spec": {
+    "triggers": [ <2>
+      {
+        "type": "github",
+        "github": {
+          "secret": "secret101"
+        }
+      },
+      {
+        "type": "generic",
+        "generic": {
+          "secret": "secret101"
+        }
+      }
+    ],
+    "source": { <3>
+      "type": "Git",
+      "git": {
+        "uri": "git://github.com/openshift/ruby-hello-world.git"
+      }
+    },
+    "strategy": { <4>
+      "type": "Source",
+      "sourceStrategy": {
+        "from": {
+          "kind": "ImageStreamTag",
+          "name": "ruby-20-centos7:latest"
+        },
+        "incremental": true
+      }
+    },
+    "output": { <5>
+      "to": {
+        "kind": "ImageStreamTag",
+        "name": "origin-ruby-sample:latest"
+      }
+    },
+  },
+}
+----
+
+<1> This specification will create a new buildConfig named ruby-sample-build.
+<2> You can specify a list of triggers, which causes a new build to be created.
+The `*imageChange*` trigger creates a new build each time the Docker image specified in the `*strategy*` `*from*` field changes. The `*github*`  and `*generic*` triggers starts the build when a callback is made from the GitHub or other source code repository hosting service, indicating  change in the source code specified in `*source*` section.
+<3> The `*source*` section define the source code repository location. You can provide additional options, such as `*sourceSecret*` or `*contextDir*` here.
+<4> The `*strategy*` section describes the build strategy used to execute the build. You can specify `*Source*`, `*Docker*` and `*Custom*` strategies here. This specification use the `*ruby-20-centos7*` Docker image that Source-To-Image will use for the application build
```

Comment:
Source, Docker *or* Custom
s/specification use/specification uses
s/Source-To-Image/Source2Image r Source-2-Image since we're using that almost everywhere from what I see